src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Button/Buttons.xaml.cs

```diff
@@ -19,7 +19,8 @@ public Buttons()
 		{
 			this.InitializeComponent();
 
-			DataContext = _viewModel = new ButtonsViewModel();
+			Loading += (s, e) => DataContext = _viewModel = new ButtonsViewModel();
```

Comment:
Is this a workaround?

Reply:
Somehow. If the DataContext stays set, then the event registrations are kept alive, causing the view to stay alive.